### DISCUSSION TITLE
facing "Hydration failed" issue when i add <Toaster/> in my app/layout.js page in Next.js (14.2.0-canary.33)

### DISCUSSION THREAD

**Discussion by abhishek9473** [shadcn-ui/ui#3070] [General]
when i use `<Toaster/>` in app/layout.js page it show me hydration error 

i mention the file below

error show - Expected server HTML to contain a matching `<div>` in `<body>`.

app/layout.js file -
import { Inter } from "next/font/google";
import "./globals.css";
import ReactQueryClientProvider from "@/providers/ReactQueryClientProvider";
import { Toaster } from "@/components/ui/toaster";

const inter = Inter({ subsets: ["latin"] });

export const metadata = {
  title: "ResumeFits",
  description: "resumefits homepage",
};

export default function RootLayout({ children }) {
  return (
    `<html lang="en">`
      `<ReactQueryClientProvider>`
        `<body className={inter.className}>`
          {children}
        `<Toaster/>`
        `</body>`
      `</ReactQueryClientProvider>`
    `</html>`
  );
}

**Comment by nazim1996** [comment]
I also facing the same issue

**Reply by abhishek9473** [reply]
make sure when you use nested layout inside main layout.
then you are not use any html or body tag.

in layout return function, use only `<div>`{children}`<div>`.
